contrib/storage-mongo/src/main/java/org/apache/drill/exec/store/mongo/plan/MongoPluginImplementor.java

```diff
@@ -292,7 +297,11 @@ public GroupScan getPhysicalOperator() {
       newSpec, columns, runAggregate);
   }
 
-  public static int direction(RelFieldCollation fieldCollation) {
+  private boolean hasMongoGroupScan(RelNode node) {
```

Comment:
@vvysotskyi I have an idea, could we transform that function as a base or interface function (such the `hasXXXGroupscan()`)? Ensure the new format based on this framework is needed to implement this filter function.

Reply:
Thanks, done.